### ISSUE TITLE
Segfault on MacOS since upgrading to Ventura

### ISSUE THREAD

**Issue by slava-vishnyakov** [xinntao/Real-ESRGAN-ncnn-vulkan#48]
I recently upgraded to MacOS Ventura and the binary no longer works. 

I tried building it against latest Vulkan SDK and XCode 14.2, but it still segfaults. 

Here's the segfault and crash data: https://github.com/slava-vishnyakov/Real-ESRGAN-ncnn-vulkan/actions/runs/5387126680/jobs/9778021348#step:6:23

Here are the changes necessary to build against latest VulkanSDK 
https://github.com/xinntao/Real-ESRGAN-ncnn-vulkan/compare/master...slava-vishnyakov:Real-ESRGAN-ncnn-vulkan:master?expand=1#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R67-R99

Thank you for your work on this!

**Comment by redthing1** [comment]
I have this as well.

**Comment by redthing1** [comment]
@slava-vishnyakov I looked at your github action and it is failing?

**Comment by redthing1** [comment]
Here is some more info on the crash:
```
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BAD_ACCESS (code=1, address=0x1)
    frame #0: 0x00000001003b6cd8 realesrgan-ncnn-vulkan`___lldb_unnamed_symbol11335 + 172
realesrgan-ncnn-vulkan`___lldb_unnamed_symbol11335:
->  0x1003b6cd8 <+172>: strb   wzr, [x0, #0x1]
    0x1003b6cdc <+176>: add    x0, sp, #0x18
    0x1003b6ce0 <+180>: bl     0x10096ce8c               ; symbol stub for: strlen
    0x1003b6ce4 <+184>: cmn    x0, #0x10
Target 0: (realesrgan-ncnn-vulkan) stopped.
(lldb)
```

**Comment by slava-vishnyakov** [comment]
@redthing1 Yes, I merely reproduced the bug in Github Actions so that authors of Real-ESRGAN-ncnn-vulkan can figure out what's wrong. I have no idea how to fix it, unfortunately. I thought that maybe newer versions of Vulkan SDK and XCode could help, but they didn't.

**Comment by yangweijie** [comment]
> I have this as well.

+1
